draft-davidben-tls-merkle-tree-certs.md

```diff
@@ -538,7 +538,7 @@ struct {
 
 struct {
     SubjectType subject_type;
-    opaque subject_info_hash[hash.length];
+    opaque abridged_subject_info<0..2^16-1>;
```

Comment:
Is there a reason this is a (length prefixed) array? If I'm not mistaken, the `abridged_subject_info` has a fixed length implied by the CA parameters hash length, so don't see a need for the length prefix. I also don't see a reason to put multiple elements.

Reply:
For TLS it's fixed length at the moment. For other subjects types it might not be fixed length; and for TLS it might even change, see #38.

Reply:
Ah, thanks for clarification!